compiler-utils/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/ContextAware.kt

```diff
@@ -112,10 +113,9 @@ interface ContextAware {
         return annotationType.resolve().isKotlinInjectQualifierAnnotation()
     }
 
+    @OptIn(KspExperimental::class)
     private fun KSType.isKotlinInjectQualifierAnnotation(): Boolean {
-        return declaration.annotations.any {
-            it.annotationType.resolve().declaration.requireQualifiedName() == qualifierFqName
-        }
+        return declaration.isAnnotationPresent(Qualifier::class)
```

Comment:
This fixes the issue and will pass CI. if you'd prefer the original implementation I can also do more debugging to see why it doesn't like `javax.inject.Scope`

Reply:
No, I think this is better. The new experimental function didn't exist when this project started. Why you ran into the other issue in the first place I cannot tell.